babel/numbers.py

```diff
@@ -56,10 +115,25 @@ def get_currency_symbol(currency, locale=LC_NUMERIC):
     >>> get_currency_symbol('USD', locale='en_US')
     u'$'
 
-    :param currency: the currency code
-    :param locale: the `Locale` object or locale identifier
+    :param currency: the currency code.
+    :param locale: the `Locale` object or locale identifier.
     """
-    return Locale.parse(locale).currency_symbols.get(currency, currency)
+    return Locale.parse(locale).currency_symbols.get(
+        currency, currency)
```

Comment:
Super minor nitpick, but no need to re-wrap this line :)

We have max-line-length 120 in the project.